samples/certs/README.md

```diff
@@ -24,6 +24,8 @@ The following `make` targets are defined:
 
 - `make root-ca`: this will generate a new root CA key and certificate.
 - `make $NAME-certs`: this will generate all needed files to bootstrap a new Citadel for cluster `$NAME` (e.g., `us-east`, `cluster01`, etc.).
```

Comment:
Can you change this one to $NAME-int-ca-certs? To make it more explicit.

Reply:
+1 - where NAME-cacerts can be a cluster name or 'vm-cacerts' for the intermediary used for VMs. 
If we want multiple VM pools using separate intermediary - we can use vmfoo-ca-certs.

Also the comment should be more clear - "will generate the intermediary certs for a cluster or vms". ( all needed files is both too broad and inaccurate).